code/addons/a11y/src/a11yRunner.ts

```diff
@@ -37,29 +45,55 @@ const runNext = async () => {
   runNext();
 };
 
-export const run = async (input: A11yParameters = defaultParameters) => {
+export const run = async (input: A11yParameters = DEFAULT_PARAMETERS) => {
   const { default: axe } = await import('axe-core');
 
-  const { element = 'body', config = {}, options = {} } = input;
-  const htmlElement = document.querySelector(element as string) ?? document.body;
+  const { config = {}, options = {} } = input;
 
-  if (!htmlElement) {
-    return;
+  // @ts-expect-error - the whole point of this is to error if 'element' is passed
+  if (input.element) {
+    throw new ElementA11yParameterError();
+  }
+
+  const context: ContextSpec = DEFAULT_CONTEXT;
+
+  if (input.context) {
+    // 1. if context exists, but it's not an object with include or exclude, it's an implicit include to be used directly
+    if (
+      !(
+        typeof input.context === 'object' &&
+        ('include' in input.context || 'exclude' in input.context)
+      )
+    ) {
+      context.include = input.context as ContextProp;
+    } else if (typeof input.context === 'object' && 'include' in input.context) {
+      // 2. if context.include exists, use it
+      context.include = input.context.include as ContextProp;
+    }
+
+    // 3. if context.exclude exists, merge it with the default exclude
+    if (
+      typeof input.context === 'object' &&
+      'exclude' in input.context &&
+      input.context.exclude !== undefined
+    ) {
+      context.exclude = (DEFAULT_CONTEXT.exclude as any).concat(input.context.exclude);
+    }
```

Comment:
If we didn't wanted to exclude the internal Storybook elements like `.sb-wrapper` - because they are excluded by default because they have `display: none` - then we wouldn't need this logic at all and could just pass in the context directly to axe.

I'm fine with either.